packages/radio-player/src/assets/img/right.ts

```diff
@@ -0,0 +1,17 @@
+import { html } from 'lit-html';
+
+const image = html`
+  <svg height="13" viewBox="0 0 8 13" width="8" xmlns="http://www.w3.org/2000/svg">
```

Comment:
question: when the svgs are used, are you wrapping them with an `img` tag to define its usage?

Reply:
You can just stick them in the DOM from here and they get used wherever you put them. They're like their own self-contained image tags.